src/platforms/python/guides/aws-lambda/index.mdx

```diff
@@ -52,19 +61,12 @@ sentry_sdk.init(
 
 The timeout warning is sent only if the timeout in the Lambda Function configuration is set to a value greater than one second.
 
-<Alert level="info" title="Note"><markdown>
-
-If you are using another web framework inside of AWS Lambda, the framework might catch those exceptions before we get to see them. Make sure to enable the framework specific integration as well, if one exists. See [_Integrations_](/platforms/python/#integrations) for more information.
-
-</markdown></Alert>
-
-<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
-
 ## Behavior
 
 With the AWS Lambda integration enabled, the Python SDK will:
 
-- Automatically report all exceptions from your Lambda Functions
+- Automatically report all exceptions and transactions from your Lambda Functions
+- You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in [Distributed Tracing](/performance-monitoring/distributed-tracing/#traces-transactions-and-spans)
```

Comment:
Note - the content for this setting is forthcoming in PR 2376.

Reply:
ok let's not merge this PR before that one then. It seems that even with the current version of #2376 this will 404, so we'll have to re-review.

Reply:
I took in the latest and the link still looks valid. @PeloWriter do you want me to update anything specific?

Reply:
This link may be more helpful. So perhaps edit to:

- You can control the the representative sample of transactions you send to Sentry by [filtering transaction event](/platforms/javascript/configuration/filtering/#filtering-transaction-events)